content/documentation/staging/tutorial/02-install-travels-demo/02-design-travels-demo.adoc

```diff
@@ -0,0 +1,58 @@
+---
+title: Review the demo application
```

Comment:
This chapter explains quite in details what is the travels demo, so perhaps it could have a more explicit title, something like "Understanding the demo application" or "Explaining the demo application" ?